<!--REQUEST_ID=963d872c-82d8-46cd-91a4-8d1bda174a75-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47965&attempt_id=9fb4d5b1-1adc-4967-9db5-8728886ba2a3)

Attempted: zstd

<details><summary>Partial log (click to expand)</summary><p>

```
--rm    : remove source file(s) after successful de/compression
 -k     : preserve source file(s) (default)
 -h/-H  : display help/long help and exit
roundTripTest: ./datagen -g512K  |  ../programs/zstd -v |  ../programs/zstd -d
./playTests.sh: line 24: md5: command not found
make[1]: *** [Makefile:335: test-zstd] Error 127
make[1]: Leaving directory '/private/tmp/nix-build-zstd-1.3.6.drv-0/source/tests'
make: *** [Makefile:73: shortest] Error 2
builder for '/nix/store/21lknfx4hlvavlxrs2cby179x0dqdgmk-zstd-1.3.6.drv' failed with exit code 2
error: build of '/nix/store/21lknfx4hlvavlxrs2cby179x0dqdgmk-zstd-1.3.6.drv' failed
```
</p></details>

--- a/pkgs/tools/compression/zstd/default.nix
+++ b/pkgs/tools/compression/zstd/default.nix
@@ -5,10 +5,10 @@
 
 stdenv.mkDerivation rec {
   name = "zstd-${version}";
-  version = "1.3.5";
+  version = "1.3.6";
 
   src = fetchFromGitHub {
-    sha256 = "0fpv8k16s14g0r552mhbh0mkr716cqy41d2znyrvks6qfphkgir4";
+    sha256 = "1x4a8r4m68m93q52ffxvaip9nnwx9yqvd1m90v80777c11s20a4n";
     rev = "v${version}";
     repo = "zstd";
     owner = "facebook";
@@ -21,8 +21,11 @@ stdenv.mkDerivation rec {
   ];
 
   checkInputs = [ file ];
-  doCheck = false; # fails with "zstd: --list does not support reading from standard input"
-                   # probably a bug
+  doCheck = true;
+  preCheck = ''
+    substituteInPlace tests/playTests.sh \
+      --replace 'MD5SUM="md5 -r"' 'MD5SUM="md5sum"'
+  '';
 
   installFlags = [
     "PREFIX=$(out)"

